File: FirstBitSolutions/C Module/Assignments/Assignment 6/Func_Assignment_1/FunctionsType4/Ques3.c
//3. Write a program to check whether a given year is a leap year.
#include <stdio.h>
int LeapYear(int);//Declaration
void main()
{
	int year;
	year =  2020;
	int res=LeapYear(year);//Call
	if(res)
	{
		printf("It is leaf Year");
	}
	else
	{
		printf("It is not leaf Year");
	}
}//Main ends here
int LeapYear(int a){
	return a%4==0&&a%100!=0||a%400==0;
}//LeapYear ends here

File: FirstBitSolutions/C Module/CtoJava Module/Assignment 1/Ques3.java
class Ques{

public static void main(String args[]){
	int year;
	year =  2020;
	
	if(year%4==0&&year%100!=0||year%400==0)
	{
		System.out.printf("It is leaf Year");
	}
	else
	{
		System.out.printf("It is not leaf Year");
	}         }
}